.travis.yml

```diff
@@ -20,7 +20,7 @@ before_install:
         export CPLUS_INCLUDE_PATH=/usr/include/python3.4m;
       fi
     - if [ $TRAVIS_PYTHON_VERSION == "3.5" ]; then
-        export CPLUS_INCLUDE_PATH=/opt/python/3.5.5/include/python3.5m;
+        export CPLUS_INCLUDE_PATH=/opt/python/3.5.6/include/python3.5m;
```

Comment:
New python version is used in Travis

Reply:
PR #58 has been merged, please pull latest commits.

Reply:
Done